test/utils.jl

```diff
@@ -11,3 +11,9 @@ end
     @test s[1:2] == "ab"
     @test s[1:2:end] == "aaaab"
 end
+
+@testitem "seval" begin
+    m = Py(Main)
+    @test m.seval("1 + 1") === 2 # Basic behavior
+    @test m.seval("1 + 1\n ") === 2 # Trailing whitespace
+end
```

Comment:
Could you move the test to https://github.com/JuliaPy/PythonCall.jl/blob/5f56a9b96b867a9f6742ab1d1e2361abd844e19f/test/jlwrap.jl#L189 please?

Reply:
Also these tests are failing because `m.seval(...)` returns a `Py`.